crane-cloud/backend
# Store transaction record on success

# Description

- store transaction record on success

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

- https://trello.com/c/cDQZtOvi

## How Can This Be Tested?

- Fetch the branch `store-transaction-record-on-success`, and run migrations and db upgrade
- using the endpoint /transactions, make a `POST` request with the record details;
`{
  "amount": 54600,
  "currency": "UGX",
  "email": "admin@cranecloud.io",
  "flutterwave_ref": "FLW-MOCK-597ae423f1470309edcb5879e3774bfa",
  "name": "admin crane",
  "phone_number": "08102909304",
  "status": "successful",
  "transaction_id": 495000,
  "tx_ref": "titanic-48981487343MDI0NzMx"
}`

- The record should be successfully added to the database, transaction_records table

## Checklist:
